src/ecdsa/serialized_signature.rs

```diff
@@ -272,9 +272,7 @@ mod tests {
     #[test]
     fn iterator_ops_are_homomorphic() {
         let mut fake_signature_data = [0; MAX_LEN];
-        // fill it with numbers 0 - 71
         for (i, byte) in fake_signature_data.iter_mut().enumerate() {
-            // up to MAX_LEN
```

Comment:
This was meant as justification why `as` is OK. :)

Reply:
Oh, I didn't read it as such, my bad. I can re-work this and improve instead of remove the comment.

Reply:
Both options sound fine to me.

Reply:
It should be on the same line as the `as` so that it's visible with grep.

rustfmt might fuck it up; see the most recent rustfmt fight in which inline comments were described as "weird" and "C like". But hopefully not.

Reply:
Well, that's at least uncommon. And you can give grep context. Although I do think it'd be nice if Rust folks were more *welcoming* towards people who need to document their code this way. ;)

Reply:
It stays on the line.